model/riscv_vmem_tlb.sail

```diff
@@ -33,112 +70,101 @@ type num_tlb_entries : Int = 64
 type tlb_index_range = range(0, num_tlb_entries - 1)
 
 // PRIVATE
-register tlb : vector(num_tlb_entries, option(TLB_Entry)) = [
-  None(), None(), None(), None(), None(), None(), None(), None(),
-  None(), None(), None(), None(), None(), None(), None(), None(),
-  None(), None(), None(), None(), None(), None(), None(), None(),
-  None(), None(), None(), None(), None(), None(), None(), None(),
-  None(), None(), None(), None(), None(), None(), None(), None(),
-  None(), None(), None(), None(), None(), None(), None(), None(),
-  None(), None(), None(), None(), None(), None(), None(), None(),
-  None(), None(), None(), None(), None(), None(), None(), None(),
-]
+register tlb : vector(num_tlb_entries, option(TLB_Entry)) = vector_init(None())
 
 // Indexed by the lowest bits of the VPN.
-function tlb_hash(vaddr : bits(64)) -> tlb_index_range =
-  unsigned(vaddr[17 .. 12])
+function tlb_hash forall 'v, is_sv_mode('v) . (
+  sv_mode : int('v),
+  vpn     : bits('v - pagesize_bits),
```

Comment:
bits('v - pagesize_bits) seems to be used quite a lot, should there be a named alias?

Reply:
Good point, I added `vpn_bits` for that.